README.md

```diff
@@ -1,4 +1,4 @@
-# EXtra-pasha
+# pasha
 
 pasha (**pa**rallelized **sha**red memory) provides tools to process data in a parallelized way with an emphasis on shared memory and zero copy. It uses the map pattern similar to Python's builtin map() function, where a callable is applied to potentially many elements in a collection. To avoid the high cost of IPC or other communication schemes, the results are meant to be written directly to memory shared between all workers as well as the calling site. The current implementations cover distribution across threads and processes on a single node.
```

Comment:
"in a parallelized way" -> "in parallel"?

It's not wrong at present, but it could be a bit simpler.

Reply:
For a similar reason, I'd drop the word 'potentially'.